src/functional/ffront/foast_passes/iterable_unpack.py

```diff
@@ -0,0 +1,98 @@
+import copy
+
+import functional.ffront.field_operator_ast as foast
+from eve import NodeTranslator, traits
+from functional.ffront.foast_passes.utils import compute_assign_indices
+
+
+class UnpackedAssignPass(NodeTranslator, traits.VisitorWithSymbolTableTrait):
+    """
+    Explicitly unpack assignments.
+
+    Example
+    -------
+    def foo():
+        a0, b0 = (1, 2)
+        __tuple_tmp_0 = (1, 2)
+        a1 = __tuple_tmp_0[0]
+        b1 = __tuple_tmp_0[1]
+        return (a1, b1)
+    """
+
+    unique_tuple_symbol_id: int = 0
+
+    @classmethod
+    def apply(cls, node: foast.FieldOperator) -> foast.FieldOperator:
+        node = cls().visit(node)
+        return node
+
+    def visit_FunctionDefinition(self, node: foast.FunctionDefinition, **kwargs):
+        new_params = self.visit(node.params, **kwargs)
```

Comment:
```suggestion
```
See below. I think this just confuses the reader of what the focus of this function is.